frontend/src/lib/types.ts

```diff
@@ -1,3 +1,90 @@
+export interface ContributeDataType {
+  projects: ProjectDataType
+  issues: IssuesDataType
+  chapters: ChapterDataType
+  committees: CommitteeDataType
+}
+
+export const contributeMockData: ContributeDataType = {
```

Comment:
This should be in the `__tests__/data` directory

Reply:
I'll make the changes